content/en/docs/tasks/administer-cluster/out-of-resource.md

```diff
@@ -341,4 +341,10 @@ to prevent system OOMs, and promote eviction of workloads so cluster state can r
 The Pod eviction may evict more Pods than needed due to stats collection timing gap. This can be mitigated by adding
 the ability to get root container stats on an on-demand basis [(https://github.com/google/cadvisor/issues/1247)](https://github.com/google/cadvisor/issues/1247) in the future.
 
+### active_file memory is not considered as available memory
+
+Currently, kubelet considers active_file (# of bytes of file-backed memory on active LRU list) as not reclaimable. For I/O intense workload, this might trigger pod eviction due to memory pressure. There is an open discussion: [https://github.com/kubernetes/kubernetes/issues/43916](https://github.com/kubernetes/kubernetes/issues/43916)
```

Comment:
I'd word this as:
```suggestion
On Linux, the kernel tracks the number of bytes of file-backed memory on active LRU list as the
`active_file` statistic. The kubelet treats `active_file` memory areas as not reclaimable.
For workloads that make intensive use of block-backed local storage, including ephemeral
local storage, kernel-level caches of file and block data means that many recently accessed
cache pages are likely to be counted as `active_file`. If enough of these kernel block buffers are
on the active LRU list, the kubelet is liable to observe this as high resource use and taint the
node as experiencing memory pressure - triggering Pod eviction.  
For more more details, see
[https://github.com/kubernetes/kubernetes/issues/43916](https://github.com/kubernetes/kubernetes/issues/43916).
```

Reply:
Thank you for the better explanation. I will take your advice.